src/components/Footer.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import Link from 'next/link';
+import Image from 'next/image';
+import MouseEvent from 'react';
+import Logo from '@/images/logo-lg.svg';
+import CurveLine from '@/images/footer-curve-line.svg';
+
+export function Footer() {
+
+  const onContextMenuHandler = (e: MouseEvent) => {
+    if(e.preventDefault != undefined)
+      e.preventDefault();
+    if(e.stopPropagation != undefined)
+      e.stopPropagation();
+    return false;
+  };
+
+  return (
+    <footer className='mt-18 py-13 pb-14 gap-5 bg-grey flex flex-col justify-center items-center'>
+      <div className='footer-title' onContextMenu={onContextMenuHandler}>
+        <Image src={Logo} className='object-cover object-center w-[200px] sm:w-[240px] md:w-[240px] lg:w-[245px] xl:w-[250px]' alt='Journey Bites' priority />
+      </div>
+      <div className='footer-curve-line mt-[-6px] mb-3 lg:mb-3.5' onContextMenu={onContextMenuHandler}>
+        <Image src={CurveLine} alt='' />
+      </div>
```

Comment:
[Required]  
1. 沒用到的 className 要移除
2. 直接用 Image 就好，外面不用再多一層 div (改了之後上面 onContextMenuHandler 要改成 HTMLImageElement)
```suggestion
       <Image src={Logo} className='object-cover object-center w-[200px] sm:w-[240px] md:w-[240px] lg:w-[245px] xl:w-[250px]' onContextMenu={onContextMenuHandler} alt='Journey Bites' priority />
        <Image src={CurveLine} className='mt-[-6px] mb-3 lg:mb-3.5' onContextMenu={onContextMenuHandler} alt='' />
```

Reply:
已排入我的待修改清單 👌
（但是是在「Figma校對設計稿」之後）

Reply:
alt 都要寫不要留空字串

Reply:
@CoolizzLuo 所以 footer curve line 它的<Image> 的 alt 也要寫 'Jouney Bites' ？

Reply:
「為 <Image> 的 alt屬性都加上值」這個程式碼改動
 commit message **prefix** 要寫什麼啊？

Reply:
要寫出這個圖片是幹嘛，像是 logo 就寫 logo 方便 SEO 閱讀，也方便在圖片沒有顯示的時候有文字知道這是做什麼
改動的話寫 `fix` 就好了

Reply:
那 footer curve line 的 alt 該寫什麼呢？
那個純粹是裝飾啊

Reply:
就寫個  wave 或 curve 之類都可以

Reply:
> https://github.com/Journey-bites/journey-bites-website/pull/17#discussion_r1623304628
> "alt 都要寫不要留空字串"

➜ 已在 2024/6/02 09:42 的 commit : c205505  修改完畢

Reply:
> "alt 都要寫不要留空字串"

➜ 已在 2024/6/02 09:42 的 commit : c205505  修改完畢